lib/acquia-http-hmac/rack_authenticate.rb

```diff
@@ -23,15 +23,20 @@ def call(env)
         return unauthorized if auth_header.empty?
 
         attributes = Acquia::HTTPHmac::Auth.parse_auth_header(auth_header)
+
+        # Pass the id to later stages
+        # Replaced id here to be able to log invalid requests too
+        env['ACQUIA-HTTP-HMAC-ID'] = attributes[:id]
```

Comment:
@pdrakeweb closing this PR in favor of this: https://github.com/acquia/http-hmac-ruby/pull/15